docs/guide/conditional-fields.md

```diff
@@ -123,31 +337,63 @@ add: {
     type: 'integer',
     min: 1,
     max: 5,
-    // 👇 Two conditions that both must be met
+    // 👇 Two conditions that both must be met (AND logic)
     if: {
       seenMovie: true,
       'votingStillOpen()': true
     }
   }
 }
 ```
+
+::: info AND vs OR Logic
+By default, multiple conditions use AND logic - all must be true. For OR logic (any condition can be true), use the `$or` operator described below.
+:::
+
 ## Conditional field requirement
 
-In addition to conditionally displaying a field, you can also conditionally mark a field as `required: true` based on the value of another field using the `requiredIf` setting. Like `if`, this property takes an object with keys *matching the names of other fields in the same schema*. The condition values must match the sibling field values *exactly* to pass.
+In addition to conditionally displaying a field, you can also conditionally mark a field as `required: true` based on the value of another field using the `requiredIf` setting. Like `if`, this property takes an object with keys *matching the names of other fields in the same schema*. The condition values must match the sibling field values *exactly* to pass, or you can use MongoDB-style comparison operators.
 
 Also like the `if` setting, the `requiredIf` can take complex conditionals with a mix of comparisons to other schema fields within the same modal, and calls to a method. All conditions must be met before the field will be active.
 
 You can have both an `if` and `requiredIf` with different conditions on the same field. If the conditions for the `if` are not met, the `requiredIf` will be ignored.
 
+```javascript
+// Example using MongoDB-style operators in requiredIf
+add: {
+  customerType: {
+    label: 'Customer Type',
+    type: 'select',
+    choices: [
+      { label: 'Individual', value: 'individual' },
+      { label: 'Business', value: 'business' }
+    ]
+  },
+  annualRevenue: {
+    label: 'Annual Revenue',
+    type: 'integer'
+  },
+  taxId: {
+    label: 'Tax ID Number',
+    type: 'string',
+    requiredIf: {
```

Comment:
Note: `requiredIf` following values feature probably doesn't work. It was out of scope in the current sprint.

Reply:
Sorry, I thought you said that it applied to all conditionals.

Reply:
Updated - this works, but `following` likely doesn't and won't be documented.